### PR TITLE
Create ceph rados gateway example

### PR DESCRIPTION
**Pull request by fabioabreureis**
I would like to create a small example for using rados gateway as S3 object store. 

closes https://github.com/vmware-tanzu/velero-plugin-example/issues/43

### CODE REVIEW

File: site/docs/master/contributions/ceph-radosgateway.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
+### 1. Object Store environment
```
Comment (skriss) [inline site/docs/master/contributions/ceph-radosgateway.md:1]: It'd be helpful to add a brief introduction explaining what this document covers, similar to e.g. https://github.com/vmware-tanzu/velero/blob/master/site/docs/master/contributions/oracle-config.md#introduction

File: site/docs/master/contributions/ceph-radosgateway.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+```
+[root@ceph-mon1 ~]# s3cmd mb s3://velero
+Bucket 's3://velero/' created
+```
+
+2. Velero Install
+
+CLI Download
+```
+$ wget https://github.com/vmware-tanzu/velero/releases/download/v1.2.0/velero-v1.2.0-linux-amd64.tar.gz
```
Comment (skriss) [inline site/docs/master/contributions/ceph-radosgateway.md:120]: It'd be nice to update this to use velero v1.4.0 if possible

Comment (fabioabreureis) [reply site/docs/master/contributions/ceph-radosgateway.md:120]: Sure, I will update this , let me test in my environment and confirm for you!

Comment (nrb) [reply site/docs/master/contributions/ceph-radosgateway.md:120]: @fabioabreureis Were you able to get this updated?

Comment (fabioabreureis) [reply site/docs/master/contributions/ceph-radosgateway.md:120]: Hi ! My appologies for my reploy so late. Yes, I will validate for velero 1.4 as recommended and update the document. 

Thanks.

**Comment by ashish-amarnath** [comment]
@fabioabreureis Just checking in to see if you are able to address the open review comments.

**Comment by ashish-amarnath** [comment]
@fabioabreureis I am going to close this PR. Please reach out once you are able to address the comments from the review.

**Pull request closed without merging**